### DISCUSSION TITLE
I can't connect Home Assistant via Huawei Solar, although it works correctly via Fusion Solar.

### DISCUSSION THREAD

**Discussion by DaradiciLevente** [wlcrs/huawei_solar#597] [Setting up the integration] [answered]
Hi.
I am quite new to Home Assistant.
I managed to integrate the SUN2000 5KTL inverter via Fusion Solar in Home Assistant, but the data comes with a delay of up to 10 minutes.
I bought a dongle and the installer installed it. I told him to set the inverter so that I can connect via Huawei Solar from Home Assistant. The installer says that everything is now set in the inverter as it should be. But I can't connect at all via Huawei Solar. Any help is welcome! Thank you!

P.S.
I mention that I use the Home Assistant version:
Korea
2024.1.3
supervisor
2023.12.1
Operating System
Front end
20240104.0

**Comment by JFHCFC** [comment]
installation guide:

https://github.com/wlcrs/huawei_solar

you have installed Huawei Solar Integration 1.3.1 ?

**Reply by DaradiciLevente** [reply]
Yes. I had this integration installed.

**Comment by DaradiciLevente** [comment]
I finally succeeded. This page helped me solve the problem: https://github.com/wlcrs/huawei_solar/wiki/Confirmed-firmware#confirmed-firmware-versions
The problem was with the slave, which should be 16, not 1 or 0, as I saw in several tutorials.

I have another problem that I haven't been able to solve, on the Power Meter I have 5 entities that don't display anything. How can this be solved?

![image](https://github.com/wlcrs/huawei_solar/assets/10895718/c8a9b584-9cfc-4adc-9903-5862e76905ad)

**Comment by JFHCFC** [comment, marked as answer]
entities are most likely disabled, click and enable

**Reply by DaradiciLevente** [reply]
Thanks! It works! Maximum respect!
